docs/Slider.md

```diff
@@ -1,4 +1,6 @@
-# Slider
+# Slider ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
+
+:warning: **The Product Price block has been deprecated in favor of the [Slider Layout](https://vtex.io/docs/app/vtex.slider-layout) app**. Although support for this component is still granted, we strongly recommend you to update your store theme with the Slider Layout's blocks in order to keep up with the component's evolution.
```

Comment:
```suggestion
:warning: **The Slider block has been deprecated in favor of the [Slider Layout](https://vtex.io/docs/app/vtex.slider-layout) app**. Although support for this component is still granted, we strongly recommend you to update your store theme with the Slider Layout's blocks in order to keep up with the component's evolution.
```

Reply:
Although support for this component is still given, it's strongly recommended that you update your store theme with the Slider Layout's blocks in order to keep up with the component's evolution.